piwik/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM quay.io/continuouspipe/php7-apache
+
+MAINTAINER Alex Biddle <alex.biddle+cp-dockerfiles@inviqa.com>
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y php-geoip php-dev libgeoip-dev && apt-get clean
+
+RUN rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+RUN echo "extension=geoip.so" >> /etc/php/7.0/apache2/php.ini
```

Comment:
I don't think a confd template is needed. If a user wants to configure some geoip settings (if there are any), then they could just add their own custom file (as with any other extension)

However, adding to php.ini I don't recommend. Instead it should use the conf.d folder for dynamic insertion.